scripts/process_cao.sh

```diff
@@ -33,14 +33,14 @@ main() {
 
     if [ ! -e "$TSV_TSNE_IN" ]
     then
-        wget "https://cells.ucsc.edu/mouse-organogenesis/tSNE.coords.tsv.gz" -O "$TSV_TSNE_IN.gz"
+        wget "https://cells.ucsc.edu/mouse-organogenesis/tSNE.coords.tsv.gz" -O "$TSV_TSNE_IN.gz" --no-check-certificate
         gunzip -df "$TSV_TSNE_IN"
     fi
 
     META_TSV_IN="$INPUT/cao.meta.tsv"
     if [ ! -e "$META_TSV_IN" ]
     then
-        wget "https://cells.ucsc.edu/mouse-organogenesis/meta.tsv" -O "$META_TSV_IN"
+        wget "https://cells.ucsc.edu/mouse-organogenesis/meta.tsv" -O "$META_TSV_IN" --no-check-certificate
```

Comment:
What happened here?

Reply:
It seemed their security certificate expired.  Not sure why - this is the error:

```
ERROR: cannot verify cells.ucsc.edu's certificate, issued by '/C=US/ST=MI/L=Ann Arbor/O=Internet2/OU=InCommon/CN=InCommon RSA Server CA':
  Issued certificate has expired.
To connect to cells.ucsc.edu insecurely, use `--no-check-certificate'.
```